db/migrate/20230209204308_add_address_to_employees.rb

```diff
@@ -0,0 +1,7 @@
+class AddAddressToEmployees < ActiveRecord::Migration[7.0]
+  def change
+    add_column :employees, :address, :string
+    add_column :employees, :state, :string
```

Comment:
Do you want a field for `city` in this table too?

Reply:
Yes a city would be good that slipped my mind thank you.